### PR TITLE
QA Review Updates

### PR DESCRIPTION
**Pull request by CatFreed**
Updating things based on Gerard's review. He wanted a couple commands and to not see warnings/error at startup.

Adding:
- `less` command
- `hexdump` command (`hd` short command)
- `dd` command (needed by urandom)
- `hostname` command (and setting the hostname to "Kubos" instead of the default "buildroot")
- CONFIG_UNIX98_PTYS - Was throwing the /dev/pts mount error at startup
- CONFIG_LOCALVERSION - This will be displayed with `uname` and is more readable than the datestamp.

Updating:
- rcS/rcK scripts to ignore errors from `ls` command. This was getting thrown when the '/home/system/etc/init.d' directory was empty because there weren't any user scripts yet.
- Upgrade partition size. Want to be able to store at least three *.itb files (current, previous, base)

Removing:
- RTC options - In theory there is a real-time timer on the AT91SAM9G20 processor which can be used as an RTC. However, it has not be fully implemented (and, more importantly, tested), so I am removing the unused/unneeded commands and configuration info, rather than add in what is missing.

Note: There are some config options outside of this list which appear to have been changed/moved. They haven't actually. I'm assuming there's some discrepancy between the generated configuration files and what was copy/pasted previously as far as presence/ordering goes.

### CODE REVIEW

**Comment by CatFreed** [comment]
[Linux additions: tail, less, ascii editor, hex viewer and/or editor](https://trello.com/c/BkPAFH0o/56-linux-additions-tail-less-ascii-editor-hex-viewer-and-or-editor)

**Comment by CatFreed** [comment]
[Try to fix or remove urandom and bin/hostname errors during linux init](https://trello.com/c/pFnBhCQb/57-try-to-fix-or-remove-urandom-and-bin-hostname-errors-during-linux-init)

File: board/kubos/at91sam9g20isis/linux-kubos.config [hunk 1]
```diff
@@ -1,3 +1,4 @@
+CONFIG_LOCALVERSION="-KubOS-1.0.0"
```
Comment (plauche) [inline board/kubos/at91sam9g20isis/linux-kubos.config]: Not right now, but at some point we should look into sourcing this version info either from a more central file or from the latest git tag.

Comment (CatFreed) [inline board/kubos/at91sam9g20isis/linux-kubos.config]: Definitely. Hard coding is never the ideal solution.

**Pull request merged**